cypress/src/plugins/removeVideosForPassedTests.tsx

```diff
@@ -0,0 +1,16 @@
+import del from 'del';
+
+const removeVideosForPassedTests = (on: Cypress.PluginEvents, _config: Cypress.PluginConfigOptions): void => {
```

Comment:
I think this function could be named a bit more accurately as it's not removing the videos at the time it's being called - it's just setting the removal mechanism up.

Reply:
Good point! I renamed the function.